libraries/opensk/src/api/persist/keys.rs

```diff
@@ -103,6 +103,16 @@ make_partition! {
     /// The stored large blob can be too big for one key, so it has to be sharded.
     LARGE_BLOB_SHARDS = 2000..2004;
 
+    /// Stored friendly names for enrolled fingerprints.
+    #[cfg(feature = "fingerprint")]
+    FRIENDLY_NAMES = 2031..2036;
```

Comment:
Let's not leave an empty key if we don't need to. If we want to reserve it, we can `_RESERVED_FRIENDLY_NAME = 2036` like for credentials.
```suggestion
    FRIENDLY_NAMES = 2031..2037;
```

Reply:
Makes sense, done.